tests/pulses/bug_tests.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+from qctoolkit.pulses.table_pulse_template import TablePulseTemplate
+from qctoolkit.pulses.function_pulse_template import FunctionPulseTemplate
+from qctoolkit.pulses.sequence_pulse_template import SequencePulseTemplate
+from qctoolkit.pulses.repetition_pulse_template import RepetitionPulseTemplate
+from qctoolkit.pulses.multi_channel_pulse_template import AtomicMultiChannelPulseTemplate
+
+from qctoolkit.pulses.plotting import plot
+
+
+class BugTests(unittest.TestCase):
+
+    def test_plotting_two_channel_function_pulse_after_two_channel_table_pulse_crash(self) -> None:
+        """ successful if no crash -> no asserts """
+        template = TablePulseTemplate(entries={'A': [(0, 0),
+                                          ('ta', 'va', 'hold'),
+                                          ('tb', 'vb', 'linear'),
+                                          ('tend', 0, 'jump')],
+                                    'B': [(0, 0),
+                                          ('ta', '-va', 'hold'),
+                                          ('tb', '-vb', 'linear'),
+                                          ('tend', 0, 'jump')]}, measurements=[('m', 0, 'ta'),
+                                                                               ('n', 'tb', 'tend-tb')])
+
+        parameters = {'ta': 2,
+                      'va': 2,
+                      'tb': 4,
+                      'vb': 3,
+                      'tc': 5,
+                      'td': 11,
+                      'tend': 6}
+        _ = plot(template, parameters, sample_rate=100, show=False, plot_measurements={'m', 'n'})
```

Comment:
Doesnt the bug happen during rendering?
We cannot use plot without emulating a display in Travis CI.